addons/binding/org.openhab.binding.yeelight/build.properties

```diff
@@ -4,7 +4,6 @@ bin.includes = META-INF/,\
                .,\
                OSGI-INF/,\
                ESH-INF/,\
-               lib/yeelight.jar,\
                about.html,\
                about_files/
```

Comment:
I'd assume the about_files can be removed and the notice about the external lib can be removed from about.html as well.